models/shared.lua

```diff
@@ -25,7 +25,7 @@ local function check_privs(meta, player)
 	local shared = " " .. meta:get_string("shared") .. " "
 	if name == meta:get_string("owner") then
 		return true
-	elseif shared:find(" " .. name .. " ") then
+	elseif shared:find(" " .. name .. " ",1,true) then
```

Comment:
please add spaces after commas -> , 1, true

Reply:
I love it when people give me some space 👍 :D